tekton-listener/pkg/apis/pipelineexperimental/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -0,0 +1,268 @@
+// +build !ignore_autogenerated
+
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code generated by deepcopy-gen. DO NOT EDIT.
```

Comment:
Would you mind adding a .gitattributes file so Github knows to suppress generated files in diffs?
https://github.com/tektoncd/pipeline/blob/master/.gitattributes

Reply:
Added a .gitattributes, ty ty